src/hardwaresecuritymodules/README.md

```diff
@@ -0,0 +1,54 @@
+Microsoft Azure CLI 'hardwaresecuritymodules' Extension
+==========================================
+
+### How to use ###
+Install this extension using the below CLI command
+```
+az extension add --name hardwaresecuritymodules
```

Comment:
The extension name is long, suggest to put hyphen between words, use `hardware-security-modules`.

Reply:
Done